everest.yaml

```
@@ -1,5 +1,5 @@
 - Name: CanyonHelper
-  Version: 1.1.4
+  Version: 1.1.5-dev
```

```suggestion
  Version: 1.1.4
```
You probably don't want to change the mod's `everest.yaml` in a contribution - with multiple PRs, this leads to conflicts (albeit tiny ones, but certainly unecessary). The version is bumped before release.